### PR TITLE
Dat 2507

### PR DESCRIPTION
**Pull request by RafLeszczynski**
Limit tile set collection in create map modal to 49 element

### CODE REVIEW

File: extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php [hunk 1]
```diff
@@ -11,7 +11,10 @@ class WikiaMapsMapController extends WikiaMapsBaseController {
 	 * @return Array
 	 */
 	public function getTileSets() {
-		$params = [];
+		$params = [
```
Comment (nandy-andy) [inline extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php]: Do we want to limit `tilesets` results always? 1) If the user creates a custom map and a list of tilesets is presented to her and 2) if the user filters presented tilesets list with a keyword?

Comment (RafLeszczynski) [inline extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php]: AFAIK Zach wanted to always display 50 latest tile set also when using search. I set it up to 49 cause the first item i the list is upload button and it makes 50 items together with it

Comment (RafLeszczynski) [inline extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php]: @nandy-andy do you think I should add a test is we stay with 49 always?

Comment (nandy-andy) [inline extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php]: It's hard to say no to tests :japanese_ogre: It's up to you to decide. You could add a test checking if there are `sort`, `limit` parameters in `$params` always and if there is the additional one `search` element when passed with the request.

But again: it's up to you. I might be guessing it's additional value for you to spend some time with PHPUnit and a test is always an additional value for our code quality.

Comment (RafLeszczynski) [inline extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php]: True :-) will do it

Comment (nandy-andy) [inline extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php]: Sweet! :sweet_potato: Good luck and have fun :smile_cat:

File: extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php [hunk 2]
```diff
@@ -11,7 +11,10 @@ class WikiaMapsMapController extends WikiaMapsBaseController {
...
+			'sort' => 'desc',
+			'limit' => 49
```
Comment (nandy-andy) [inline extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php]: Oh, one more thing. We might want to add a comment line here why it is `49` - the thing about the upload button :) Extracting it to a const may be an overkill if it's being used only in this one place... on the other hand it's a good practice to keep "magic numbers" in const. I'd ask one or few more experienced developers what's their opinion about it :neckbeard:

File: extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php (hunk 1 above)
Comment (jacekjursza) [inline extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php]: IMO the reason to write unit test here would be to 'freeze' values of parameters 'sort' and 'limit'. But after talking to Rafał I believe there is no reason to do that - as the service back-end will handle it anyway and there is no risk of braking something easily. ;) 

On the others hand - I would move '49' number to const of this class and describe it - why it's 49.

**Comment by jacekjursza** [comment]
LGTM

**Comment by nandy-andy** [comment]
LGTM :smile:

**Pull request merged**